app/models.py

```diff
@@ -0,0 +1,133 @@
+# This is where you define the models of your application.
+# This may be split into several modules in the same way as views.py.
+
+from datetime import datetime
+from werkzeug.security import generate_password_hash, \
+     check_password_hash
+from flask_login import UserMixin
+from itsdangerous import (TimedJSONWebSignatureSerializer
+                          as Serializer, BadSignature, SignatureExpired)
+
+from app import app, create_app
+
+db = create_app()
+
+
+class User(db.Model, UserMixin):
+    """this is the person model"""
+    __tablename__ = "user"
+    id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String(255), nullable=False, unique=True)
+    password = db.Column(db.String(128), nullable=False)
+    bucketlist = db.relationship('Bucketlist', backref='user', lazy='dynamic')
+
+    def __init__(self, email, password):
+        """initialize with name."""
+        self.email = email
+        self.set_password(password=bytes(str(password), 'utf-8'))
+        self.password = self.pwd_hash
+
+    def set_password(self, password):
+        self.pw_hash = generate_password_hash(password)
+
+    def check_password(self, password):
+        return check_password_hash(self.pw_hash, password)
+
+    def generate_auth_token(self, expiration=600):
+        s = Serializer(app.config['SECRET_KEY'], expires_in=expiration)
+        return s.dumps({'id': self.id})
+
+    @staticmethod
+    def verify_auth_token(token):
+        s = Serializer(app.config['SECRET_KEY'])
+        try:
+            data = s.loads(token)
+        except SignatureExpired:
+            return None  # valid token, but expired
+        except BadSignature:
+            return None  # invalid token
+        user = User.query.get(data['id'])
+        return user
+
+    def __repr__(self):
+        return "<User: {}>".format(self.email)
+
+    def save(self):
```

Comment:
I'm surprised as to why you didn't make use of this function here in your routes.